src/SqlPersistence.Tests/Outbox/OutboxPersisterTests.cs

```diff
@@ -148,4 +149,63 @@ public void StoreAndGet()
         return await persister.Get(messageId, null);
     }
 
+    [Test]
+    public async Task StoreAndCleanup()
+    {
+        using (var connection = await dbConnection.OpenConnection())
+        {
+            for (var i = 0; i < 13; i++)
+            {
+                await Store(i, connection);
+            }
+        }
+
+        await Task.Delay(1000);
```

Comment:
What do you think about passing `Func<DateTime>` to `OutboxPersister` in `ctror` to make this tests not time-dependent?